ansible/requirements.yml

```diff
@@ -4,3 +4,4 @@
 - src: stackhpc.monasca-rsyslog
 - src: stackhpc.grafana-conf
 - src: stackhpc.gluster-cluster
+- src: stackhpc.os-manila-mount
```

Comment:
This change should be removed from this PR as it is was already committed as part of `multinet-ohpc` branch.